src/components/Header.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Link } from "react-router-dom"
+import logo from '../assets/kpedditLogo1.png'
+
+export default function Header (){
+
+    return (
+        <>
+        <div className="header-grid">
```

Comment:
You should use a semantic tag here - `header` would be good. This is `div` heavy.